packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1517,7 +1517,10 @@ export class ContainerRuntime
 		electedSummarizerData: ISerializedElection | undefined,
 		chunks: [string, string[]][],
 		dataStoreAliasMap: [string, string][],
-		runtimeOptions: Readonly<Required<IContainerRuntimeOptions>>,
+		runtimeOptions: Readonly<
+			Required<Omit<IContainerRuntimeOptions, "flushMode" | "enableGroupedBatching">> &
```

Comment:
@jason-ha, @anthony-murphy, @Josmithr -- I'm moving forward with merging this, but to be pedantic, I don't think this kind of change is in general perfectly backwards compatible. Wanted to call it out to check my work and for transparency.

If someone is using `typeof ContainerRuntime` to type a class declaration, their implementation may theoretically depend on `flushMode` being `Required`, but then after this change a caller could stop passing it.  Here's a [TS Playground example](https://www.typescriptlang.org/play/?#code/MYewdgzgLgBANiA5jAvDUkRwKYDoGIDcAUMcHAIYQQwBCV2MA3sTGzAPQcwDyAMgBEAXOnDQATgFdgUEOIAUAMxAgRTCiLCSAtgCNs4gDQxdmnfvEBfAJTNLrdlxgA5AKIB1IQ7YYJ02QrKqswaMFp6Bsa6APxmEVa2TPb2ZGKwPFAAFgYAwpTUIlAAngAO2CCKdAyo6Pk0LOyc3ACqENg0WYxYACYwEACWiGAUUJLi2MaSwwDuFOMwWSMw-bCZVOhrYIjY3d6ikFBSMnJKKmqh4RZRcRY2zHuNvrDqUZY1QSSNjQTyFFHWn3YyRIxCcOQocDg-S2om6jH6NDWJRKRQWIBgIG0KxMJgYvXAC2yYWw0wWpWhyAqVTaxDAJN4nXEeSoEHkTBgoQAjDAbCQgA).